### DISCUSSION TITLE
Fans connected to Commander Core not controllable

### DISCUSSION THREAD

**Discussion by joen05** [EvanMulawski/FanControl.CorsairLink#50] [Q&A] [answered]
So I've installed this plugin and it detected my commander core as well as the fans connected to it, but I can only control the pump itself, and not the fans.  Is there anything that I need to do to get the fans to be controllable?  Their RPM is detected in the "speed" section.

**Comment by EvanMulawski** [comment]
Hi @joen05, is the speed sensor paired with the control? You might need to pair them using the "Pair speed sensor..." option in the fan's menu (three dots).

![image](https://github.com/EvanMulawski/FanControl.CorsairLink/assets/1044791/2ad6b59b-bec3-43f7-a385-de19a4dc6e60)

**Reply by joen05** [reply]
I actually don't see that option for the fan.  They don't show up in the Controls Section, only in the Speed section.

**Reply by EvanMulawski** [reply]
Can you post a screenshot of what you're seeing in Fan Control?

**Reply by joen05** [reply]
<img width="1430" alt="image" src="https://github.com/EvanMulawski/FanControl.CorsairLink/assets/5625467/3bb4af7a-d409-417b-b3ff-e5488ef4dd57">

This is what I see.

**Reply by EvanMulawski** [reply]
Enable "Show hidden cards" in the top left menu; then you should be able to pair the fans with the speed sensors.

**Reply by joen05** [reply]
Ok - that seemed to get everything to appear.  I'm sure this is common sense, but does iCue need to be completely removed for the fan curves in fan control to be applied?

**Reply by joen05** [reply]
And actually, it seems to be stuck pairing the speed sensor - how long does that take?

**Reply by joen05** [reply]
"I tried automatically pairing the speed sensors and it gave me a "no speed sensor reacted to Commander core fan #" error.

**Reply by EvanMulawski** [reply, marked as answer]
You'll likely need to pair it manually - not using the automatic option. When you select "Pair speed sensor", click the icon of the speed sensor with the matching fan number.

iCUE must be closed to use this (you may also need to stop the "Corsair Service" service and set its startup type to Manual using the Windows Services control panel).
